provisioner/README.md

```diff
@@ -85,6 +85,12 @@ controllerinstall: true
 # forces ansible.workshops collection to install latest edits every time
 developer_mode: true
 
+# SHA value of targeted AAP bundle setup files.
+provided_sha_value: ea2843fae672274cb1b32447c9a54c627aa5bdf5577d9a6c7f957efe68be8c01
+
+# Automation controller install setup command. Default: "./setup.sh -e gpgcheck=0" if undefined or empty
+controller_install_command: './setup.sh -e gpgcheck=0'
```

Comment:
i would move this to the defaults/main.yml for the control node role to simplify this for users